### ISSUE TITLE
Integration stopped working

### ISSUE THREAD

**Issue by waltershimoon** [jbnunn/Alexa-MyQGarage#31]
Hi, I was able to get everything work great.  It worked for months.  Recently, perhaps the last 1 to 2 months, it has stopped working.  Has something changed?

**Comment by tigerbrain** [comment]
Hi Walter,

There has been a lot of changes in February.  Please update your code based on the latest code posted in the repository.

Thanks!

tigerbrain

**Comment by waltershimoon** [comment]
I just redid everything per the new instructions.  I get back "there was a problem with the requested skills response".

**Comment by tigerbrain** [comment]
Hi Walter,

Please make sure that you are not URL encoding the username and password.  These values should be typed in exactly the way you type them in using the MyQ app.

Hope this helps, thanks!

tigerbrain

**Comment by waltershimoon** [comment]
Thanks.  as of now, UN and PW are in plain English, surrounded by " ".  Still no luck.  I will start over again.  Let me know if you can think of anything else.

**Comment by waltershimoon** [comment]
I've started over completely.  Still same error.  With regards to the lambda-upload.zip file :  do I create a zip file with main.py and the two folders, or do I move the contents of the folders into the top level of the zip hierarchy?

**Comment by tigerbrain** [comment]
Hi Walter,

Just wanted to make sure that your UN and PW is only surrounded by a single double quote.  "abc@abc.com" not ""abc@abc.com""

inside the zip file should be main.py and the two folders with all its contents. 

Did you get a chance to look at the stepbystepinstruction document?  May want to do that to make sure that you didn't miss anything.

**Comment by tigerbrain** [comment]
Hi Walter,

You should also upload the results of your lambda logs to aid us in troubleshooting your issue better.  The way to get to the logs is mentioned in the main page of the repository immediately below the brief installation instructions.

Thanks 

tigerbrain

**Comment by waltershimoon** [comment]
Hi, I really appreciate your help on this.

Yes, UN & PW match that format.

I've followed the stepbystep exactly.

This should work.....

I'll restart one more time, just in case.. then upload logs if it fails still.

Thanks!!
WS

**Comment by waltershimoon** [comment]
Fail.

Here is what seems to be going wrong, found in the logs:

Unable to import module 'lambda_function': No module named lambda_function

**Comment by tigerbrain** [comment]
Hi Walter, 

In the step by step guide, focus on the section that talked about naming your function main.lambda_handler, maybe that is where your issue is. 

Hope you get this working this time.

tigerbrain

**Comment by waltershimoon** [comment]
Well... it didn't work.

I scrapped it and went for the Multidoor solution instead. that worked no problem.  Its not ideal, because I only have 1 door.....

**Comment by tigerbrain** [comment]
Hi Walter,

I was about to give you that suggestion myself since it is easier to install.  I did code the multidoor version to support single door installations so you can essentially just tell Alexa "ask garage door to close"

**Comment by waltershimoon** [comment]
Yeah, I saw that. I love it. Thanks.

I am trying to set this all up for a friend who actually has 2 garages.  Right now, when I reference Door 1 or Door 2, it only controls 1 garage (SMALL).  He has them named BIG GARAGE and SMALL GARAGE.  any thoughts?

**Comment by tigerbrain** [comment]
Hi Walter,

Unfortunately, this will require a modification to the main.py code, something that I am currently unable to do because of my busy schedule.

Sorry

tigerbrain

**Comment by tigerbrain** [comment]
Hi Walter,

In the multidoor repository, I noticed that there is a pending pull request contributed by rcohn that may just work for you by adding "the big door" and "the small door" to DOOR_NUMBERS.  You may want to just give that a try, though it won't be perfect, it should work.

**Comment by tigerbrain** [comment]
Hi Walter,

You may ignore my last suggestion as it will not work without modifying main.py code.

Sorry for the confusion

**Comment by waltershimoon** [comment]
No problem. Thanks for all your help on this.

**Comment by jbnunn** [comment]
Thanks @tigerbrain and @waltershimoon
